content/en/docs/collector/resiliency.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Resiliency
+description: How to configure a resilient OTel Collector pipeline
+---
+
+The OpenTelemetry Collector is designed with components and configurations to
+minimize data loss during telemetry processing and exporting. However,
+understanding the potential scenarios where data loss can occur and how to
+mitigate them is crucial for a resilient observability pipeline.
+
+## Understanding Collector Resilience
+
+The Collector's resilience primarily revolves around how it handles data when
+the configured endpoint (the destination for traces, metrics, or logs) becomes
+unavailable or when the Collector instance itself experiences issues like
+crashes.
```

Comment:
Could we reformulate this a bit to describe what we mean by resilience? For example, is it fair to say that a resilient Collector is one that reduces data loss and crashes under stress or high load? It might be more relatable that way.

Reply:
I added the following prior to this paragraph:

> A resilient Collector maintains telemetry data flow and processing capabilities
> even when facing adverse conditions, ensuring the overall observability pipeline
> remains functional.